bindings/python/vara-feature/pybind_Feature.cpp

```diff
@@ -45,6 +42,10 @@ void init_feature_module_feature(py::module &M) {
       .def("__hash__", &vf::Feature::hash)
       .def_property_readonly("name", &vf::Feature::getName,
                              R"pbdoc(The name of the feature.)pbdoc")
+      .def(
+          "is_root",
+          [](vf::Feature &F) { return llvm::isa<vf::RootFeature>(F); },
```

Comment:
```suggestion
          [](const vf::Feature &F) { return llvm::isa<vf::RootFeature>(F); },
```
I, think this should be const, otherwise, we could get a problem when we call python functions on an underlying const object.